blocks/library/pullquote/index.js

```diff
@@ -95,9 +102,11 @@ registerBlockType( 'core/pullquote', {
 
 		return (
 			<blockquote className={ `align${ align }` }>
-				{ value && value.map( ( paragraph, i ) => <p key={ i }>{ paragraph.props.children }</p> ) }
+				{ value && value.map( ( paragraph, i ) =>
+					<Editable.Value tagName="p" key={ i }>{ paragraph }</Editable.Value>
```

Comment:
With `Editable.Value`, do you think we can get rid of this map now as well and automatically key?

Reply:
Would also make more sense as in `edit` the `Editable` covers the array, but here in `save` `Editable.Value` only covers an item in that array.

Reply:
notice the comment above `// Need a better matcher joining the values`, we can't do this right now because we can't write a matcher extracting all the `p` elements into a single string.

We used to be able to use `.join` when declaring the `source` but this is not possible since the source is declarative only. Maybe worth adding a `div` wrapper, it makes sense to me or come up with a `concat` matcher: `value: `concat( `p`, prop( 'outerHTML' ) )`

Reply:
Sorry didn't see the comment

Reply:
Yeah, it's not that obvious that it's related. Anyway, the comment helps to surface the issue here.

Reply:
Do you get the sense that in most of the cases we'd want a joined set of HTML, we'd be better served by supporting block nesting instead (i.e. should we bother with a matcher for this at all?).

Reply:
> Do you get the sense that in most of the cases we'd want a joined set of HTML, we'd be better served by supporting block nesting instead (i.e. should we bother with a matcher for this at all?).

I don't think so, I think would have the same issue even with nested blocks, the hard part is that we're not able to target (or filter) nodes where we miss some structure (no wrapper around the similar `p`)